src/modules/dashboard/components/ExpenditurePage/Stages/LinkedMotions/LinkedMotions.css

```diff
@@ -50,17 +50,6 @@
   color: var(--pink);
```

Comment:
why is there no corresponding change in css.d.ts file? Not committed?

Reply:
I'm not sure if you mean deleting the link class? If so, hover state on this class still remains, so the css.d.ts file has not changed.

Reply:
Disregard my comment. `.link:hover` is left, so of course there shouldn't be any changes in d.ts file.